keyboards/donutcables/budget96/readme.md

```diff
@@ -0,0 +1,44 @@
+# budget96
+
+96 key atmega32a keyboard. 
+
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
+Hardware Supported: Budget96 PCB  
+Hardware Availability: [DonutCables](https://donutcables.com/)  
+
+Make example for this keyboard (after setting up your build environment):
+
+    make donutcables/budget96:default
+
+Flashing
+
+**Reset Key:** Hold down the key located at `K00`, commonly programmed as left control while plugging in the keyboard. You may also hold down the key located at `K50`, commonly programmed as the `Esc` key. 
+
+ps2avr(GB) boards use an atmega32a microcontroller and a different bootloader. It is not flashable using the regular QMK methods. 
+
+Windows: 
```

Comment:
Are there Linux instructions for this? Seems strange to me to include Windows and Mac, but not Linux. (I think the other ps2avrGB readmes are similar; perhaps the template needs an update?)

Reply:
I made the decision to use macos and windows as most people in our community have those two systems. I'm also making the assumption that if you're using Linux, you most likely know what package manager to use.